source/features/sticky-sidebar.tsx

```diff
@@ -8,7 +8,7 @@ import features from '.';
 
 const deinit: VoidFunction[] = [];
 // Both selectors are present on conversation pages so we need to discriminate
-const sidebarSelector = pageDetect.isRepoRoot() ? '.repository-content .flex-column > :last-child [data-pjax]' : '#partial-discussion-sidebar';
+const sidebarSelector = pageDetect.isRepoRoot() ? '.repository-content .flex-column > .flex-shrink-0 > [data-pjax]' : '#partial-discussion-sidebar';
```

Comment:
The selector for the repo root page was broken on latest Firefox. Fixed it but it's still brittle, should be use the `select().closest()` strategy?

Reply:
Yes please if possible

Reply:
Ah with selector-observer it's a little more complicated.

I think in that case the best option would be to select the sidebar with the shortest selector and then exclude items in its listener. Maybe.